Makefile

```diff
@@ -1,10 +1,10 @@
 .PHONY: build deploy
 
-build:
+build: 
 	go install ./cmd/manager
 
 build-image: 
-	operator-sdk build kabanero-operator:latest
+	operator-sdk build kabanero/kabanero-operator:latest
```

Comment:
As a developer using these build targets, it would be nice if the image name could be replaced by a variable or something.  On the other hand, running `operator-sdk build imagename` is not a hard thing to do manually.  The build-image target will work great for when we're updating the public image out on dockerhub.

I don't think there needs to be a variable right now, it would be nice to have later.

Reply:
I added a varible